### PR TITLE
Implement New Design for Call To Action

### PR DESCRIPTION
**Pull request by moollaza**
WIP

/cc @loganom

### CODE REVIEW

**Comment by loganom** [comment]
@moollaza I've finished updating this project. I was a little curious about the scss/duckduckhack/_*.scss files though, as it appears that they aren't being used to generate the style.css file when the default task is ran (and I had no success with the build task either). This could be user error, but i'm not entirely sure. 

I had originally made the changes in style.css which is why they can be found there.

File: src/css/style.css [hunk 1]
```diff
@@ -10658,3 +10678,15 @@ img {
...
+.header-image {
+  width: 144px;
+  height: 144px;
+  margin-bottom:20px;
+}
+
+.sub-image {
+  width: 108px;
+  height: 108px;
+  margin-bottom:20px;
```
Comment (moollaza) [inline src/css/style.css]: can we move this to `ddh--sub__image` above and remove this class?

Then we can apply this class to all 3 images and ensure they have a bottom margin 👍

**Comment by moollaza** [comment]
@loganom excellent work, this is looking great. I noticed that only the double-rocket image ("Becoming a Leader") has bottom margin though. Can we update the CSS/HTML so that all the images have some space between the text and image?

Other than that, this looks good for merge 👍 

> I was a little curious about the scss/duckduckhack/_*.scss files though, as it appears that they aren't being used to generate the style.css file when the default task is ran (and I had no success with the build task either). This could be user error, but i'm not entirely sure.

Oh wow, looks like you're right. I'm not familiar with the history here, but I would expect us to process the scss into css, combine it with other css and minify it all together for production.

Would you be interested in implementing those changes? This could be a task for another day.

**Comment by moollaza** [comment]
Ping! Would you be able to finish this one up? We'd love to get this live!

**Comment by loganom** [comment]
@moollaza updated the sub-image class

**Comment by moollaza** [comment]
@loganom Thanks! This LGTM :+1:

**Pull request merged**